sonar-project.properties

```diff
@@ -0,0 +1,21 @@
+sonar.projectKey=bossan_django-keycloak
```

Comment:
Probably needs to be changed to your sonar or this file should be deleted completely.

Reply:
Will do